### PR TITLE
LibGfx: Verify reserved ICC header bytes are zero

### PR DESCRIPTION
**Pull request by nico**
…and some even smaller tweaks.

### CODE REVIEW

File: Userland/Libraries/LibGfx/ICCProfile.cpp [hunk 1]
```diff
@@ -234,15 +234,20 @@ ErrorOr<XYZ> parse_pcs_illuminant(ICCHeader const& header)
     return xyz;
 }
 
+template<size_t N>
+bool all_bytes_are_zero(const u8 (&bytes)[N])
+{
+    for (u8 byte : bytes)
+        if (byte != 0)
+            return false;
```
Comment (linusg) [inline Userland/Libraries/LibGfx/ICCProfile.cpp:242]: Please add curly braces to multi-line `for`

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibGfx/ICCProfile.cpp (hunk 1 above)
Comment (nico) [reply Userland/Libraries/LibGfx/ICCProfile.cpp]: Done. Did we request a clang-format feature for this?

Comment (linusg) [reply Userland/Libraries/LibGfx/ICCProfile.cpp]: Not that I know of, would be nice to have!

Comment (nico) [reply Userland/Libraries/LibGfx/ICCProfile.cpp]: Filed https://github.com/llvm/llvm-project/issues/59864.

**Pull request merged**